### DISCUSSION TITLE
Log_to_metric convert mcs to seconds in field

### DISCUSSION THREAD

**Discussion by Manniac07** [vectordotdev/vector#20306] [Q&A]
Is my field syntax correct. Or is there a better way to convert mcs to second
wfm_scheduleViewerAction_latency-prod:
type: log_to_metric inputs:
- filter_wfm_scheduleVtewerAction_prod_message
metrics:
- type: histogram
field: http.request.request_time_mcs/1000000
name: nice_apache_access_agent_adherence increment_by_value: true namespace: vector

**Comment by jszwedko** [comment]
Unfortunately, the configuration for `log_to_metric` is static; you can't have an expression for the `field` like you have. The workaround would be to use `remap` to divide the value before feeding it into `log_to_metric`.

**Reply by Manniac07** [reply]
@jszwedko Like this?

format_wfm_scheduleViewerAction_test_Request_time_mcs:
inputs:
-filter_wfm_scheduleViewerAction_test_message
type: "remap"
source: . = parse_json!(.message)
      .http.request.request_time_mcs =to_float!(http.request.request_time_mcs) / 1000000
